src/encoded/docs/schema-changes.md

```diff
@@ -154,87 +154,208 @@ This new object is an array of example objects that can successfully POST agains
 Updating an existing schema
 ----------------
 
-There are 2 categories when we talk about updating an existing schema:
+There are two situations we need to consider when updating an existing schema: (1) No update of the schema version (2) Update schema version
 
-* No affect on schema version
-* Update schema version
+**When not to update a version**
 
+* Do not update the schema version if the updated schema allows all existing objects in the database to continue to validate. For example, a new enum value in an existing list of enums would not cause existing objects of that type to fail validation.
 
-**The schema version is updated, requiring an upgrade of the schema, when the objects stored in the database will not validate and POST to the updated schema.**
+**When to update a version**
 
+* Schema version has to be updated (bumped up by 1) if the change that is being introduced will lead to a potential invalidation of existing objects in the database. 
 
-### No affect on schema version
+* Examples include:
+    1) Changing the name of a property in the existing schema.
+    2) Removing a property from the existing schema.
+    3) A property that previously allowed free text is now changed to a possible list of allowed enums.
+    4) If an existing enum is removed from a property.
 
-1. In the **schemas** directory, make edits  to properties in the JSON file named after the object.
+* Most of the cases described above are examples where existing objects could potentially fail the validation under the new schema version. Hence, an additional step of adding an upgrade script is required. This will ensure that all the existing objects will be upgraded (changed) such that they will be valid under the new schema version.
+
+* **NOTE:** You should update the schema version when making substantial changes to an existing schema even if these changes do not cause existing objects using this schema to fail validation.
```

Comment:
I think I get it now, and so maybe instead of the next bullet point (“Particularly, this will be helpful…”) you could add to _this_ bullet point something like, “The new schema version number in cases like this helps submitters and users realize that a substantial change has been made to the schema.”